Mathlib/Data/ENNReal/Inv.lean

```diff
@@ -151,6 +151,20 @@ protected theorem div_pos (ha : a ≠ 0) (hb : b ≠ ∞) : 0 < a / b :=
   ENNReal.mul_pos ha <| ENNReal.inv_ne_zero.2 hb
 #align ennreal.div_pos ENNReal.div_pos
 
+@[simp] lemma mul_inv_le_one (a : ℝ≥0∞) : a * a⁻¹ ≤ 1 := by
```

Comment:
This is the same as existing `ENNReal.div_self_le_one`.

Reply:
Well it's similar, but it's not *the same*?